renderers/team_renderer.py

```diff
@@ -28,6 +28,7 @@ def render_team_details(cls, handler, team, year, is_canonical):
         social_media_future = media_query.TeamSocialMediaQuery(team.key.id()).fetch_async()
         robot_future = Robot.get_by_id_async('{}_{}'.format(team.key.id(), year))
         team_districts_future = team_query.TeamDistrictsQuery(team.key.id()).fetch_async()
+        participation_future = team_query.TeamParticipationQuery(team.key.id())._query_async()
```

Comment:
`_query_async` is some internal ndb method, you should stick to the documented interface, which you can see here: https://cloud.google.com/appengine/docs/python/ndb/queryclass

Like Eugene said, `fetch_async` is what you want here